rules/windows/defense_evasion_execution_msbuild_started_by_office_app.toml

```diff
@@ -26,33 +26,39 @@ rule_id = "c5dc3223-13a2-44a2-946c-e9dc0aa0449c"
 severity = "high"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:MSBuild.exe and
-  process.parent.name:(eqnedt32.exe or excel.exe or fltldr.exe or msaccess.exe or
-                       mspub.exe or outlook.exe or powerpnt.exe or winword.exe)
+procesds where event.type in ("start", "process_started") and
```

Comment:
```suggestion
process where event.type in ("start", "process_started") and
```

Reply:
:whew:, good catch.

This def should fail validation, must be a gap in `kql` @rw-access